crates/fuel-core/src/schema/tx.rs

```diff
@@ -262,12 +263,28 @@ impl TxStatusSubscription {
     }
 }
 
+use crate::schema::tx::types::SubmittedStatus;
+use fuel_core_storage::Error as StorageError;
+
 #[async_trait::async_trait]
 impl<'a> TxnStatusChangeState for StreamState<'a> {
     async fn get_tx_status(
         &self,
         id: fuel_types::Bytes32,
     ) -> StorageResult<Option<TransactionStatus>> {
-        types::get_tx_status(id, &TransactionQueryContext(self.db), self.txpool).await
+        let query = self.db;
+        match query
```

Comment:
Hmm, the previous code with `get_tx_status` was better=) Could we return it back?

Reply:
moved back